aur.py

```diff
@@ -20,7 +20,7 @@ def update_packages(module, tool, auronly):
     assert tool in TOOL_CMD_MAP
 
     cmd = ['env', 'LC_ALL=C'] + TOOL_CMD_MAP[tool] + ['-Su']
-    if auronly:
+    if auronly and (tool != 'yay'):
```

Comment:
This isn't what I meant -- we should fail hard (with an error) if auronly is set but we're using yay

Reply:
This simply ignores it, this is what readme describes as well. If we should hard fail, to be consistent we should also hard fail whenever recurse is set to False and state present is used or any other improper or unsupported choice. What do you think? Is it what you suggest?